tests/format/typescript/typeparams/line-breaking-after-extends-2.ts

```diff
@@ -0,0 +1,25 @@
+a = {
+  parseFunctionBodyAndFinish<
+    T extends
+      | N.Function
+      | N.TSDeclareMethod
+      | N.TSDeclareFunction
+      | N.ClassPrivateMethod,
+  >() {}
+}
+
+function parseFunctionBodyAndFinish<
+  T extends
+    | N.Function
+    | N.TSDeclareMethod
+    | N.TSDeclareFunction
+    | N.ClassPrivateMethod
+    | Foo
+    | Bar
+    | Baz
+>();
+
+function parseFunctionBodyAndFinish<
+  T extends // comment
+    N.Function | N.TSDeclareMethod | Baz
+>();
```

Comment:
Can you add code from https://github.com/prettier/prettier/issues/14808#issue-1701165776 to the test?

Reply:
[4f62476](https://github.com/prettier/prettier/pull/14858/commits/4f6247686f8b2023a8a55ba4f727d623a95e1c7b)